docs/guides/references/best-practices.mdx

```diff
@@ -327,6 +327,8 @@ are common for:
 - CMS instances, such as a Contentful or Wordpress instance.
 - Other types of services under a domain in which you control.
 
+#### Challenges with Social Connection Authentication
```

Comment:
I'm not sure how I feel about Social Connection Authentication.  

```suggestion
#### Potential Challenges Authenticating with Social Platforms
```

Reply:
suggestion reads better imo